ci/deploy.yml

```diff
@@ -80,3 +80,46 @@ msg-system deploy dev:
     - deploy
   script:
     - aws s3 cp packages/suite-data/files/message-system/config.v1.jws s3://data.trezor.io/config/develop/config.v1.jws
+
+# npm packages deploy to npm registry
+link deploy npm:
+  stage: deploy
+  only:
+    <<: *run_everything_rules
+  when: manual
+  before_script:
+    - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+  script:
+    - nix-shell --run "yarn && cd ./packages/blockchain-link && npm publish"
+  after_script:
+    - rm ~/.npmrc
+  tags:
+    - deploy
```

Comment:
Let's move all the common things to an anchor:

```
  when: manual
  before_script:
    - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
  after_script:
    - rm ~/.npmrc
  tags:
    - deploy
```

Reply:
@vdovhanych why not `tags` as well?

Reply:
I like when i can see it right away on what type of the runner it's running if you search for something. But as i think everything in the deploy.yml file should run on deploy tags so we could add this as an anchor for everything there.

Reply:
You can use this argument for basically anything :). Other people might like to see `after_script`. I would be consistent in this.

Reply:
So i should keep the tag there as we want to have it consistent with the rest of the files?

Reply:
Sorry, I meant consistency with the anchor :laughing:. Let's just add it to the anchor and I'll shut up.

Reply:
Eeegh I still do not see the change? :thinking:

Reply:
🙈  should be fixed now